sdk-lib/src/main/rust/lib.rs

```diff
@@ -1062,7 +1109,7 @@ pub unsafe extern "C" fn Java_cash_z_ecc_android_sdk_jni_RustBackend_createToAdd
 
```

Comment:
On line 1078, "account argument must be nonnegative". Also look for other incorrect uses of "positive".